fastly-edge-assignments/src/handlers/assignments.rs

```diff
@@ -0,0 +1,241 @@
+use eppo_core::configuration_store::ConfigurationStore;
+use eppo_core::eval::{Evaluator, EvaluatorConfig};
+use eppo_core::ufc::UniversalFlagConfig;
+use eppo_core::{Attributes, Configuration, SdkMetadata};
+use fastly::http::StatusCode;
+use fastly::kv_store::KVStoreError;
+use fastly::{Error, KVStore, Request, Response};
+use serde::{Deserialize, Serialize};
+use sha2::{Digest, Sha256};
+use std::collections::HashMap;
+use std::sync::Arc;
+
+#[derive(Debug, Deserialize)]
+struct RequestBody {
+    subject_key: String,
+    subject_attributes: Arc<Attributes>,
+    // TODO: Add bandit actions
+    // #[serde(rename = "banditActions")]
+    // #[serde(skip_serializing_if = "Option::is_none")]
+    // bandit_actions: Option<HashMap<String, serde_json::Value>>,
+}
+
+// Response
+
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "UPPERCASE")]
+enum AssignmentFormat {
+    Precomputed,
+}
+
+#[derive(Debug, Serialize)]
+struct Environment {
+    name: String,
+}
+
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+struct FlagAssignment {
+    allocation_key: String,
+    variation_key: String,
+    variation_type: String,
+    variation_value: serde_json::Value,
+    extra_logging: HashMap<String, serde_json::Value>,
+    do_log: bool,
+}
+
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+struct AssignmentsResponse {
+    created_at: i64,
```

Comment:
major: `i64` deviates from the tech design and how we usually represent timestamps.

We could use `chrono::DateTime<chrono::Utc>` and it would serialize to ISO timestamp:
```suggestion
    created_at: chrono::DateTime<chrono::Utc>,
```